sdk/src/storage/preview_storage_client.rs

```diff
@@ -0,0 +1,308 @@
+use momento_protos::control_client::scs_control_client::ScsControlClient;
+use momento_protos::store::store_client::StoreClient;
+use tonic::codegen::InterceptedService;
+use tonic::transport::Channel;
+
+use crate::grpc::header_interceptor::HeaderInterceptor;
+use crate::storage::messages::control::create_store::{CreateStoreRequest, CreateStoreResponse};
+use crate::storage::messages::control::delete_store::{DeleteStoreRequest, DeleteStoreResponse};
+use crate::storage::messages::control::list_stores::{ListStoresRequest, ListStoresResponse};
+use crate::storage::messages::data::get::{GetRequest, GetResponse};
+use crate::storage::messages::momento_store_request::MomentoStorageRequest;
+use crate::storage::messages::store_value::StoreValue;
+use crate::storage::preview_storage_client_builder::{
+    NeedsConfiguration, PreviewStorageClientBuilder,
+};
+use crate::storage::{Configuration, DeleteRequest, DeleteResponse, SetRequest, SetResponse};
+use crate::MomentoResult;
+
+/// Client to work with Momento Store.
```

Comment:
```suggestion
/// Client to work with Momento Storage.
```

Reply:
I want to use the product name here. Will it be called Momento Storage or Momento Store?

Reply:
Storage, Anita is right